components/picker/stories/picker.stories.js

```diff
@@ -33,34 +38,58 @@ export default {
 		labelPosition: {
 			name: "Label position",
 			type: { name: "string" },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["top", "side"],
+			control: {
+				type: "select",
+				labels: {
+					side: "side (inline start)",
+				},
+			},
+		},
+		helpText: {
+			name: "Help text",
+			description: "The help text that is displayed below the Picker. This uses a separate Help text component outside of the Picker markup.",
+			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
 				category: "Content",
 			},
-			options: ["top", "left"],
-			control: { type: "select" },
+			control: { type: "text" },
 		},
 		withSwitch: {
-			name: "Display with a switch component",
+			name: "Display Switch component",
+			description: "Displays a Switch component after the Picker. This is used for testing the vertical alignment between the side label, Picker, and Switch.",
```

Comment:
I think in this case I wouldn't want to advertise it, since it's mostly for a special case in VRTs, and the Switch is not a part of the Picker component. I don't want to hide the control, so it can still be referenced and linked to if needed.